### PR TITLE
Add Load on Demand topic for WebComponents.

### PR DESCRIPTION
**Pull request by MayaKirova**

### CODE REVIEW

**Comment by HUSSAR-mtrela** [comment]
@MayaKirova is this still a draft PR? I asked because @tishko0 labeled this PR as verified!

File: doc/en/components/grids/tree-grid/load-on-demand.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+The LoadChildrenOnDemand callback provides two parameters:
+
+- parentID - the ID of the parent row that is being expanded.
+- done - callback that should be called with the children when they are retrieved from the server. 
+
+```typescript
+public loadChildren = (parentID: any, done: (children: any[]) => void) => {
+    this.getData(parentID, (children) => done(children));
+}
+```
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/tree-grid/load-on-demand.md:52]: build flag lines 29-52 for WC in case this topic is later enabled for Blazor

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/grids/tree-grid/load-on-demand.md (hunk 1 above)
Comment (dkamburov) [reply doc/en/components/grids/tree-grid/load-on-demand.md:52]: @MayaKirova Can you take a look?

Comment (MayaKirova) [reply doc/en/components/grids/tree-grid/load-on-demand.md:52]: I don't see anything that would necessarily need to be excluded for blazor between those lines but ok.

**Pull request merged**